### PR TITLE
Note minimum NeoVim version

### LINKED ISSUE #40: `README.md` should note minimum NeoVim version
It appears that the `server_info->[...` syntax used for the RLS and `clangd` servers is not valid in NeoVim 0.1.7 (the current Debian stable version). The syntax error is hidden from the user except via the `:messages` command.

Updating to NeoVim 0.2.0, from Debian testing fixed the issue and made RLS integration work properly.

I do not see any mention of the NeoVim version requirements in the `README.md`.

### PR DESCRIPTION
**Pull request by BatmanAoD**
Resolves #40.

There are of course other ways to highlight the minimum version required; I just thought this would be the easiest and most immediately-visible way.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 asyncomplete.vim
 ================
 
-Provide async autocompletion for vim8 and neovim with `timers`.
+Provide async autocompletion for vim8 and neovim (0.2.0+) with `timers`.
```
Comment (prabirshrestha) [inline README.md]: Might be add recommended v0.2.0 or higher instead?

File: README.md [hunk 2]
```diff
@@ -106,6 +106,8 @@ endif
 
 **Refer to [vim-lsp wiki](https://github.com/prabirshrestha/vim-lsp/wiki/Servers) for configuring other language servers.** Besides auto-complete language server support other features such as go to definition, find references, renaming symbols, document symbols, find workspace symbols, formatting and so on.
 
+**Please note** that setup for certain LSP servers requires support for v0.2.0+ of neovim.
```
Comment (prabirshrestha) [inline README.md]: should be certain `asyncomplete sources` instead of `LSP servers`.

**Comment by BatmanAoD** [comment]
@prabirshrestha This is under the LSP section. Non-LSP plugins will presumably work with asyncomplete.vim for whatever NeoVim version the plugin itself works with.

**Comment by prabirshrestha** [comment]
@BatmanAoD Then we should rephrase it to `vim-lsp` requires neovim 0.2.0+ since it in not the lsp sources but the vim-lsp plugin itself.

**Comment by BatmanAoD** [comment]
@prabirshrestha Better? (vim-lsp doesn't *always* require 0.2; it's the setup itself with the lambda syntax)

File: README.md [hunk 3]
```diff
@@ -106,7 +106,7 @@ endif
 
 **Refer to [vim-lsp wiki](https://github.com/prabirshrestha/vim-lsp/wiki/Servers) for configuring other language servers.** Besides auto-complete language server support other features such as go to definition, find references, renaming symbols, document symbols, find workspace symbols, formatting and so on.
 
-**Please note** that setup for certain LSP servers requires support for v0.2.0+ of neovim.
+**Please note** that vim-lsp setup for certain LSP servers requires support for v0.2.0+ of neovim.
```
Comment (prabirshrestha) [inline README.md]: Can we change it to that`vim-lsp setup requires support for v0.2.0+ of neovim.`

**Comment by prabirshrestha** [comment]
@BatmanAoD vim-lsp always requires lambda.

**Comment by BatmanAoD** [comment]
@prabirshrestha Odd, I thought I'd reveiwed the setup for all the listed servers in the wiki and found that only `clangd` and RLS setup used the lambda, but I see now that I was mistaken.

How is this version?

**Comment by prabirshrestha** [comment]
Looks good. Thanks
